app/config/default.json

```diff
@@ -1,4 +1,5 @@
 {
+    "api": "https://www.targetvalidation.org/api",

```

Comment:
we might remove the `/api/latest/` bit from the api url, if we are successful in deploying the api in google apps. the version will be prepended to the domain, but is not there yet